flake.nix

```diff
@@ -6,67 +6,73 @@
     ds.url = "github:numtide/devshell?ref=master";
   };
   outputs = { self, np, fu, ds }:
-  with fu.lib;
-  eachSystem [ "x86_64-linux" "x86_64-darwin" ] (system:
-  let
-    version = with np.lib;
-    "${substring 0 8 self.lastModifiedDate}.${self.shortRev or "dirty"}";
-    config = { };
-    overlay = final: super:
-    let
-      souffle = with final;
-      callPackage (import ./nix/souffle.nix { pkgs = final; }) { };
-      haskellPackages = super.haskell.packages.ghc902.override {
-        overrides = hself: hsuper: {
-          souffle-haskell = with final.haskell.lib;
-          with hself;
-          (overrideCabal
-          (addBuildTools (callCabal2nix "souffle-haskell" ./. { }) [
+    with fu.lib;
```

Comment:
@luc-tielen, most of the code changes below are caused by `nixfmt` formatter. If you don't mind, let's enforce its usage prior to merging changes on nix files. What do you think?

Reply:
Yes, good suggestion. :+1: Let's do that in another PR though.